Spigot-Server-Patches/0371-Implement-furnace-cook-speed-multiplier-API.patch

```diff
@@ -0,0 +1,84 @@
+From b0b5bd713acc27765d02af599416a74d1101e90f Mon Sep 17 00:00:00 2001
+From: Tassu <git@tassu.me>
+Date: Thu, 13 Sep 2018 08:45:21 +0300
+Subject: [PATCH] Implement furnace cook speed multiplier API
+
+Signed-off-by: Tassu <git@tassu.me>
+
+diff --git a/src/main/java/net/minecraft/server/TileEntityFurnace.java b/src/main/java/net/minecraft/server/TileEntityFurnace.java
+index c439b20c..ed055098 100644
+--- a/src/main/java/net/minecraft/server/TileEntityFurnace.java
++++ b/src/main/java/net/minecraft/server/TileEntityFurnace.java
+@@ -11,6 +11,7 @@ import java.util.Map.Entry;
+ import javax.annotation.Nullable;
+ // CraftBukkit start
+ import java.util.List;
++
+ import org.bukkit.craftbukkit.block.CraftBlock;
+ import org.bukkit.craftbukkit.entity.CraftHumanEntity;
+ import org.bukkit.craftbukkit.inventory.CraftItemStack;
+@@ -29,6 +30,7 @@ public class TileEntityFurnace extends TileEntityContainer implements IWorldInve
+     private int ticksForCurrentFuel;
+     private int cookTime;
+     private int cookTimeTotal;
++    public double cookSpeedMultiplier = 1.0; // Paper - cook speed multiplier API
```

Comment:
can you move this up 2 lines, it'll help avoid unnecessary conflicts on version updates.